Source/st_lib.c

```diff
@@ -221,7 +221,7 @@ void STlib_updatePercent
   char *outrng,             //jff 2/16/98 add color translation to digit output
   int refresh )
 {
-  if (refresh || *per->n.on) // killough 2/21/98: fix percents not updated;
+  if (/*refresh || */*per->n.on) // killough 2/21/98: fix percents not updated;
```

Comment:
This looks dirty to me.

Reply:
I'm not sure about that. What was the original bug fixed here? If left unchanged, percent symbols are always displayed on-screen in overlay mode.

Reply:
I guess this has to do with the percent signs not updating when switched between gray and colored.

Reply:
I change it to `if (*per->n.on && refresh)`, seems to work.